docker/dev/xdebug.ini

```diff
@@ -0,0 +1,7 @@
+[xdebug]
+zend_extension=xdebug.so
+xdebug.mode=debug
+xdebug.start_with_request=yes
+xdebug.client_host=host.docker.internal
+xdebug.client_port=9000
```

Comment:
Absolutely no reason to override the default. Since Xdebug 3, the default is 9003, for good reason (because 9000 conflicts with FPM and many other services).